org.radare.Cutter.yml

```diff
@@ -0,0 +1,47 @@
+id: org.radare.Cutter
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: Cutter
+rename-desktop-file: Cutter.desktop
+rename-icon: cutter # I could install it under the right name, but then I need to patch the desktop file
+finish-args:
+ - --share=ipc
+ - --socket=x11
+ - --socket=wayland
+ - --filesystem=host
+
+modules:
+ - name: radare
+   # translation of the arch build script
+   sources:
+    - type: archive
+      url: https://github.com/radare/radare2/archive/2.6.0.tar.gz
+      sha256: c4722519e53bbb03289e3b98e89e637e2d1f401d02cf78a198e42c2e92aeb820
+    - type: patch
+      path: fix-capstone.patch
+    - type: patch
+      path: fix-zip.patch
+    - type: patch
+      path: fix-lib-dist.patch
+    - type: git
+      url:  https://github.com/aquynh/capstone.git
+      commit: 37569a6874c8547b349a80823adda9284499fe80
+      dest: shlr/capstone
+   config-opts:
```

Comment:
Nitpick: we usually put `sources` last.

Reply:
I've seen that, but always found it strange. You first download stuff, and then you build it. May I ask why?